modules/calib3d/src/ap3p.h

```diff
@@ -0,0 +1,76 @@
+#ifndef P3P_P3P_H
+#define P3P_P3P_H
+
+#include "precomp.hpp"
+
+class ap3p {
```

Comment:
Although this class in not public, but it would be nice to put in under "cv" namespace.

BTW, Extra indentation (four "spaces" symbols, or similar) is not needed in namespaces.